build.gradle

```diff
@@ -93,6 +158,20 @@ project(':cruise-control-core') {
 
 project(':cruise-control') {
   apply plugin: 'scala'
+  apply plugin: 'maven-publish'
+  apply plugin: 'com.jfrog.artifactory'
+
+  test.dependsOn('checkstyleMain', 'checkstyleTest')
+
+  task sourcesJar(type: Jar, dependsOn: classes) {
```

Comment:
sourcesJar and javadocJar may be easy to move to subProjects{} closure - they are identical always